test/integration/features/story_health.feature

```diff
@@ -25,7 +25,7 @@ Feature:
     @darwin @linux 
     Scenario: Check cluster health
         Given executing "crc status" succeeds
-        And stdout contains "Running (v4."
+        And stdout should match ".*Running \(v\d+\.\d+\.\d+.*\).*"
```

Comment:
Is there any reasoning  between using "should match" here and "matches" in the hunk before that one? Or both are the same?

Reply:
@cfergeau both evaluate the same function (so are the same). I have them both because it reads funny in e.g. `When stdout should match...`.